src/scenes/AcceleratorRouter/ParticipantProjects/ProjectProfileGisMaps.tsx

```diff
@@ -442,7 +442,7 @@ const ProjectProfileGisMaps = () => {
             <PlantingSiteMap
               mapData={filteredSiteData}
               style={{ width: '100%', borderRadius: theme.spacing(1) }}
-              layers={[selectedLayer || 'Planting Site']}
+              layers={[selectedLayer || 'Planting Site', 'Planting Site']}
```

Comment:
Is this change intentional?

Reply:
Yes, I wanted to always include the planting site layer (in addition to the selected layer) so that I can display the site name also in zones and subzones